lib/cli/src/commands/mod.rs

```diff
@@ -70,12 +71,21 @@ pub(crate) trait AsyncCliCommand: Send + Sync {
     type Output: Send + Sync;
 
     async fn run_async(self) -> Result<Self::Output, anyhow::Error>;
+
+    fn setup(&self) -> Result<(), anyhow::Error> {
+        ctrlc::set_handler(move || {
```

Comment:
Note: we should probably restrict this to TTYs. 

`stdin.is_terminal()`.

See https://doc.rust-lang.org/stable/std/io/trait.IsTerminal.html#tymethod.is_terminal